tests/test_orc.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+# ==============================================================================
+"""
+Test ORCDataset
+"""
+
+import os
+import numpy as np
+
+import tensorflow as tf
+import tensorflow_io as tfio
+
+
+def test_orc_input():
+    """test_pcap_input
+  """
```

Comment:
`test_orc_input`, also, I think the `"""` in ln 28 can be aligned with the previous line

Reply:
updated.. interesting, not covered by lint